nvflare/app_opt/xgboost/histogram_based_v2/sec/sec_handler.py

```diff
@@ -51,6 +52,7 @@ def error(self, fl_ctx: FLContext, msg: str):
     def _abort(self, error: str, fl_ctx: FLContext):
         self.error(fl_ctx, error)
         self.system_panic(reason=error, fl_ctx=fl_ctx)
+        XGBExecutor.notify_client_done(Constant.EXIT_CODE_JOB_ABORT, fl_ctx)
```

Comment:
See my previous comment: change to fire the XGB_ABORTED event and do not call system_panic here - let the executor's event handler call it.